docs/source/getting_started/getting_started.ipynb

```diff
@@ -45,19 +46,151 @@
+    "![MAAP Project log in](_static/loggingin1.png)\n",
+    "\n",
+    "\n",
+    "2. Click the “Login with EarthData Account” button. If this is your first time logging in, you should be redirected to an EarthData Login page that looks like this:\n",
+    "![Login with EarthData](_static/loggingin2.png)\n",
+    "\n",
+    "\n",
+    "3. Enter your “EarthData Login” account credentials here and click “Log in”. You should see a temporary page that says “Redirecting”, followed by the MAAP showing your Workspaces (which will be empty to start):\n",
+    "![MAAP Jupyter logging in](_static/loggingin3.png)"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Uploading your public SSH key \n",
+    "### Creating a workspace\n",
+    "\n",
+    "Workspaces are effectively a Jupyter \"computer in the cloud\". To get started with Jupyter you need to create a workspace.\n",
+    "\n",
+    "1. In the top-left corner of the MAAP dashboard, under “NASA MAAP”, click “Get Started”. You should see a menu that looks like this:\n",
+    "![Choose a workspace stack](_static/createws1.png)\n",
+    "\n",
+    "\n",
+    "2. Select \"Basic Stable”. This is called a \"Stack\" and represents a type of cloud compute environment that will be set up. If you are interested in seeing more about each Stack, the adjacent link in the left-hand area is where you can see the configuration of each Stack in detail. After choosing \"Basic Stable\", you will see a loading screen that looks like this – wait for it to finish loading.\n",
+    "![Loading Basic Stable](_static/createws2.png)\n",
+    "\n",
+    "\n",
+    "3. Once the workspace has loaded, you should see a  Jupyter interface that looks like this (note: You will see fewer environments and items in your root directory — this is normal! You may also see some notifications in the bottom right that look like errors about SSH Keys and other things; that is normal as well. You will also see one asking if you would like to take a guided tour.).\n",
+    "![New Workspace View](_static/createws3.png)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Jupyter Interface overview\n",
+    "When you first log in you may see a notification in the bottom right about a guided tour. Feel free to view the tour, which will give you a quick overview of the Jupyter user interface. You can also find the MAAP Tour in the Help menu at any time.\n",
+    "![MAAP Tour Notification](_static/maap_tour_notification.png)\n",
+    "\n",
+    "\n",
+    "In addition to [typical JupyterLab menu bar and sidebar](https://jupyterlab.readthedocs.io/en/stable/user/interface.html) configuration:\n",
+    "\n",
+    "#### MAAP Jupyter Menus\n",
+    "![Jupyter Menus](_static/interface1.png)\n",
```

Comment:
This image includes "MAAP Login" at the top which I believe we are removing (gone in DIT). Also, MAAP login isn't addressed in the list even though it isn't a "typical Jupyterlab menu bar"

Reply:
@rtapella update the Jupyter menu screenshot